### PR TITLE
fix: InvalidQuery error due to feature flags

### PR DESCRIPTION
**Pull request by shuplenkov**
## Issue being fixed or feature implemented

"InvalidQueryError: Invalid query" error during testnet sync process

## What was done?

Do not fetch feature flag contract id on the height it was created 

## How Has This Been Tested?

Unit tests & sync with testnet

## Breaking Changes

No

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by shumkov — APPROVED** [review]
Good catch! 👍

File: test/unit/featureFlag/getLatestFeatureFlagFactory.spec.js [hunk 1]
```diff
@@ -59,4 +59,10 @@ describe('getLatestFeatureFlagFactory', () => {
 
     expect(result).to.equal(null);
   });
+
+  it('should return null if height specified is equal the height of activation', async () => {
```
Comment (thephez) [inline test/unit/featureFlag/getLatestFeatureFlagFactory.spec.js:63]:
```suggestion
  it('should return null if height specified is equal to the height of activation', async () => {
```

**Review by thephez — APPROVED** [review]
:+1:

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**